src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js

```diff
@@ -139,6 +140,15 @@ const TriggerFormChannels = ({
           <FormikLabel text='Notify me via' />
         </SidecarExplanationTooltip>
         <div className={styles.notifyBlock}>
+          <ChannelCheckbox
+            channel={CHANNEL_NAMES.Email}
+            isActive={isActive}
+            isDisabled={isDisabled}
+            toggleChannel={toggleChannel}
+            isRequired={isRequired}
+            recheckBrowserNotifications={recheckBrowserNotifications}
```

Comment:
Why do we need to do this for EMAIL channel? @GarageInc

Reply:
Line 149? It is a wrapped component, which shows dialog with settings. Dialog not only with email, but also with push/telegram channels
![image](https://user-images.githubusercontent.com/14061779/65953759-49021a00-e44d-11e9-8149-23d4333c0e09.png)